profiler/test/Directory.Build.props

```diff
@@ -18,4 +18,16 @@
       <CheckEolTargetFramework>false</CheckEolTargetFramework>
     </PropertyGroup>
 
-</Project>
\ No newline at end of file
+    <Target Name="PackageReferenceCopyToOutput" AfterTargets="Build">
+      <ItemGroup>
+        <PackageReferenceFiles Condition="%(PackageReference.CopyToOutputDirectory) != '' AND %(PackageReference.FolderName) != ''"
+          Include="$(NugetPackageRoot)\%(PackageReference.FolderName)\%(PackageReference.Version)\%(PackageReference.CopyToOutputDirectory)" />
+      </ItemGroup>
+      <Message Condition="%(PackageReference.CopyToOutputDirectory) != ''" 
+               Text="> $(NugetPackageRoot) | %(PackageReference.FolderName) | %(PackageReference.Version) | %(PackageReference.CopyToOutputDirectory) [$(NugetPackageRoot)\%(PackageReference.FileName)\%(PackageReference.Version)\%(PackageReference.CopyToOutputDirectory) | $(OutDir)]" Importance="high"/>
+      <Message Condition="%(PackageReference.CopyToOutputDirectory) != '' AND '@(PackageReferenceFiles->'%(Identity)')' != ''"
+               Text="  Copy @(PackageReferenceFiles->'%(Identity)') to $(OutDir)"
+               Importance="high"/>
+      <Copy SourceFiles="@(PackageReferenceFiles)" DestinationFolder="$(OutDir)" />
+    </Target>
```

Comment:
Is this necessary? 🤔 IIRC, it's not (we used to have it, but removed it recently)

Reply:
oh ok, I followed an old PR.